hypertrace-core-graphql-impl/src/main/java/org/hypertrace/core/graphql/impl/GraphQlFactory.java

```diff
@@ -14,8 +14,6 @@ public static GraphQLConfiguration build(
     final Injector injector = Guice.createInjector(new GraphQlModule(config, lifecycle));
 
     return GraphQLConfiguration.with(injector.getInstance(GraphQLSchema.class))
-        .with(config.isAsyncServlet())
```

Comment:
they removed this config without declaring it a breaking change - oops! (it's no longer required, they configure sync vs async automatically based on return type of the handler.